src/commands/init.ts

```diff
@@ -188,7 +188,7 @@ export default class Init extends Command {
     ];
 
     if (warnings) {
-      this.log(boxMessage(['⚠️  Init finished with warnings:', ...warnings, '', ...body]));
+      this.log(boxMessage(['⚠️ Init finished with warnings:', ...warnings, '', ...body]));
```

Comment:
This leads to some weird looking formatting because the emoji is a bit wider than the usual monowidth character (at least in my terminal)